apps/dashboard/app/(app)/authorization/permissions/[permissionId]/delete-permission.tsx

```
@@ -64,6 +64,11 @@ export const DeletePermission: React.FC<Props> = ({ trigger, permission }) => {
     },
   });
 
+  function handleDialogOpenChange(newState: boolean) {
```

Biome

[error] This function is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.

(lint/correctness/noUnusedVariables)